spec/features/home_page_feature_spec.rb

```diff
@@ -1,14 +1,14 @@
 require 'rails_helper'
 
-RSpec.feature 'Home page content', type: :feature do
+RSpec.describe 'Home page content', type: :feature do
   let(:user) { create :user }
 
-  scenario 'Check correct redirect' do
+  it 'Check correct redirect' do
```

Comment:
@sinpai why not `it 'checks correct redirect'?`

Reply:
👍